app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -229,6 +231,15 @@ public function updateContacts(string $id, UpdateContactInfoRequest $request): R
         return Redirect::back();
     }
 
+    public function storeContact(StoreContactRequest $request, string $id): RedirectResponse
```

Comment:
Maybe best to write also a unit test for this new endpoint. It is important to test the happy path and the edge case of an unauthorized user.

Reply:
Done

[Add tests for storeContact method](https://github.com/cultuurnet/publiq-platform/pull/1017/commits/2e124236085e70bd541f1003a142dd87950f91a5)